nvm.sh

```diff
@@ -1994,6 +1995,51 @@ nvm() {
       return 42
     ;;
 
+    "self-upgrade")
+      local current_nvm_version
+      local latest_nvm_version
+      local origin_nvm_version
+
+      current_nvm_version="$(nvm --version)"
+
+      if nvm_has "git" && [ -d "$NVM_DIR/.git" ]; then
+        command cd "$NVM_DIR"
+        command git fetch --tags
+        latest_nvm_version="$(command git tag | command sort -V | command tail -n 1)"
```

Comment:
Okay I just updated to `command git describe --abbrev=0 --tags --match 'v[0-9]*' origin` like the manual upgrade method mentioned in readme.